core/arch/atomic_exports.h

```diff
@@ -733,7 +733,8 @@ atomic_dec_becomes_zero(volatile int *var)
 #        define ATOMIC_1BYTE_READ(addr_src, addr_res)               \
```

Comment:
Please update the PR description: it seems to talk about memory to memory atomic reads which is not done on any architecture in these atomic functions.

Reply:
Done.